manifest.md

```diff
@@ -137,18 +146,27 @@ Unlike the [Manifest List](#manifest-list), which contains information about a s
     Subsequent layers MUST then follow in the order in which they are to be layered on top of each other.
     The algorithm to create the final unpacked filesystem layout MUST be to first unpack the layer at index 0, then index 1, and so on.
 
-- **`annotations`** *hashmap*
+- **`annotations`** *string-string map*
 
-    This OPTIONAL property contains arbitrary metadata for the manifest list.
-    Annotations is a key-value, unordered hashmap.
-    Keys MUST be unique within the hashmap, and best practice is to namespace the keys.
-    Common annotation keys include:
-    * **created** date on which the image was built (string, timestamps type)
-    * **authors** contact details of the people or organization responsible for the image (freeform string)
-    * **homepage** URL to find more information on the image (string, must be a URL with scheme HTTP or HTTPS)
-    * **documentation** URL to get documentation on the image (string, must be a URL with scheme HTTP or HTTPS)
-    * **source** URL to get the source code for the binary files in the image (string, must be a URL with scheme HTTP or HTTPS)
+    This OPTIONAL property contains arbitrary metadata for the image manifest.
+    Annotations MUST BE a key-value map where both the key and value MUST be strings.
+    While the value MUST be present, it MAY be an empty string.
+    Keys MUST be unique within this map, and best practice is to namespace the keys.
+    Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
+    Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by subsequent specifications.
+    If there are no annotations then this property MAY either be absent or be an empty map.
+    Implementations that are reading/processing the image manifest MUST NOT generate an error if they encounter an unknown annotation key.
+
+	This specification defines the following key, which MAY be used by image manifest authors:
```

Comment:
darn - thought I got both of those... guess not. fixed - thanks!
-Doug

> On Sep 7, 2016, at 11:34 AM, W. Trevor King <notifications@github.com> wrote:
> 
> In manifest.md <https://github.com/opencontainers/image-spec/pull/164#discussion_r77847153>:
> 
> > -    Common annotation keys include:
> > -    * **created** date on which the image was built (string, timestamps type)
> > -    * **authors** contact details of the people or organization responsible for the image (freeform string)
> > -    * **homepage** URL to find more information on the image (string, must be a URL with scheme HTTP or HTTPS)
> > -    * **documentation** URL to get documentation on the image (string, must be a URL with scheme HTTP or HTTPS)
> > -    * **source** URL to get the source code for the binary files in the image (string, must be a URL with scheme HTTP or HTTPS)
> > +    This OPTIONAL property contains arbitrary metadata for the image manifest.
> > +    Annotations MUST BE a key-value map where both the key and value MUST be strings.
> > +    While the value MUST be present, it MAY be an empty string.
> > +    Keys MUST be unique within this map, and best practice is to namespace the keys.
> > +    Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
> > +    Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by subsequent specifications.
> > +    If there are no annotations then this property MAY either be absent or be an empty map.
> > +    Implementations that are reading/processing the image manifest MUST NOT generate an error if they encounter an unknown annotation key.
> > +
> > +	This specification defines the following key, which MAY be used by image manifest authors:
> “key” → “keys”. And this line has a tab indent, but should probably be indented with spaces.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/opencontainers/image-spec/pull/164/files/db887afc460b2183ecc506fb16a91f2a3c291c9d#r77847153>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AB2sXyV3w7mMLuVzm6l-xlqxz_Frsn1Tks5qntmggaJpZM4I-n6z>.
>